news/2022-09-16-project-loved-september-2022.md

```diff
@@ -0,0 +1,341 @@
+
+---
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1165592.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Slax - Too Much Loli](https://osu.ppy.sh/beatmapsets/1165592#mania) by [NikoSek](https://osu.ppy.sh/users/4088977)\
+190 – 266 BPM, 3:05 | 4.88★, 5.24★, 5.65★, 6.02★, 6.39★\
+*description written by [kadoen](https://osu.ppy.sh/users/12780575)*
+
+Did you not have enough loli [last time](https://osu.ppy.sh/beatmapsets/1126349)? Well, you can have some more... But be careful, as it may now be too much!
+
+In all seriousness, the wild popularity of Nikosek's approachable charting is too much to ignore, so it's time for the second part of their "lolilogy" to enter Loved voting. You know the drill: lots of jumpstream, some jumptrill, and a few breaks between them, all combined to create a very fun experience accessible to a wide audience.
+
+Whether you're trying for a pass, an S, or an SS — on any of the rates — click on that banner just above and make your vote count!
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1190299.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Carlito - Who's That Boy](https://osu.ppy.sh/beatmapsets/1190299#mania) by [Evening](https://osu.ppy.sh/users/2193881)\
+120 – 216 BPM, 3:15 | 5.22★ – 8.47★\
+*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*
+
+He's Carlito. Yes I've just spoiled it to you. That Boy is Carlito. But ~~crappy~~ jokes aside, Evening's *Who's That Boy* has finally been nominated! We hope that both fans of the patterning and people who just want to have a laugh will enjoy this chart as much as we did.
+
+For the unaware, this is a heavily anchor-focused chordjack chart. Its uniqueness comes not only from the song choice (which is already a plus), but rather the way it uses said anchors advantageously. Other than that, you should expect an unrelenting chart that's not going to hold any punches with its constant barrage of chords. So expect your forearms to be a little sore after trying this out.
+
+With all of that said, we hope that this legacy map piques an interest with both people who already played it and those who have yet to try this masterpiece.
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/623741.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Zekk - Skrillexore](https://osu.ppy.sh/beatmapsets/623741#mania) by [Azubeur](https://osu.ppy.sh/users/1594604)\
+175 BPM, 2:37 | 4.62★\
+*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*
+
+It's been quite some time since they've been under the spotlight ever since [*USAO's Dynamite*](https://osu.ppy.sh/beatmapsets/589727) was first featured, but Azubeur is back with *Zekk's Skrillexore!*
```

Comment:
```suggestion
It's been quite some time since they've been under the spotlight ever since [*USAO's Dynamite*](https://osu.ppy.sh/beatmapsets/589727) was first featured, but Azubeur is back with *Zekk - Skrillexore!*
```